CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - `Security` in case of vulnerabilities.
 
 ## [unreleased x.x.x] -
+### Fixed
+- Cannot call library method from Java language [#129](https://github.com/CanHub/Android-Image-Cropper/issues/129)
```

Comment:
Let's be a little more specific here, for documentation history. Could use the PR name `Make isReadExternalStoragePermissionsRequired visible in Java` I know we have two more methods with it, but not an issue

Reply:
Done.  In my defense, I just re-used the comment from line 21 ;)

Reply:
We are always improving ^^, don't feel the need to defend yourself hahaha. I make some mistakes cause no one like to review my PRs ;'( 

Still looking for people to help more =)